libtransmission/utils.c

```diff
@@ -403,6 +403,21 @@ int64_t tr_getDirFreeSpace(char const* dir)
     return free_space;
 }
 
+bool tr_canWriteFile(char const* path)
+{
+    FILE* file = fopen(path, "wb+");
+    int fopen_error = errno;
+
+    if (file != NULL)
+    {
+        fclose(file);
+        return 1;
```

Comment:
Use proper boolean literals from <stdbool.h> (already included in transmission.h), i.e.
```suggestion
        return true;
```